node_app/models/Observation.js

```diff
@@ -0,0 +1,87 @@
+/*
+Observation {
+  status : enum (final, amended, corrected)
+  category : enum (vital-signs, procedure, exam)
+  code : enum (?)
```

Comment:
I also don't know which enum values we need to provide.  Who can help us with that?

Reply:
What will these Observations represent? What clinical measurements are taken during the workflow. If it's blood pressure, for example, we can use a SNOMED code to represent that: https://snomedbrowser.com/Codes/Details/75367002

Reply:
I'm not sure. The example valueset from hl7 has over a thousand codes

Reply:
@dmcook5208 how does the adverse event documentation work? How will we allow vaccinators to document these?

Reply:
In the Immunization resource, there is this:
![image](https://user-images.githubusercontent.com/61951196/103235757-7798ca00-4908-11eb-93b9-8d6df8621dcd.png)

Right now, we have a page in the POD app for documenting adverse reactions...but this is something we should discuss with Waukesha county.  Anything a user documents in our app will also need to be documented in VAERS, so I'm not sure how beneficial this feature will really be at this point (as it is just duplicated documentation).  Here is what our "Adverse Reactions" page would currently look like - it mimics the VAERS form:
![image](https://user-images.githubusercontent.com/61951196/103235916-e83fe680-4908-11eb-8b34-57b297d0ea8a.png)